fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -538,10 +538,15 @@ public LogicalPlan visitTableFunction(TableFunctionRelation node, ExpressionMapp
                     true));
         }
 
+        FunctionCallExpr expr = new FunctionCallExpr(tableFunction.getFunctionName(), node.getChildExpressions());
+        expr.setFn(tableFunction);
+        ScalarOperator operator = SqlToScalarOperatorTranslator.translate(expr, context);
+
         Map<ColumnRefOperator, ScalarOperator> projectMap = new HashMap<>();
-        for (Expr e : node.getChildExpressions()) {
-            ScalarOperator scalarOperator = SqlToScalarOperatorTranslator.translate(e, context);
-            if (e instanceof SlotRef) {
+        for (int i = 0; i < node.getChildExpressions().size(); i++) {
+            Expr e = node.getChildExpressions().get(i);
```

Comment:
If a conversion is added here, the operator should be null and the type should be the standard, Expr has no meaning